apps/organisations/migrations/0022_ckeditor_iframes.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.20 on 2023-11-16 11:35
+
+from bs4 import BeautifulSoup
+from django.db import migrations
+from django.utils import translation
+
+
+def replace_iframe_with_figur(apps, schema_editor):
+    template = (
+        '<figure class="media"><div data-oembed-url="{url}"><div><iframe src="'
+        '{url}"></iframe></div></div></figure>'
+    )
+    Organisation = apps.get_model("a4_candy_organisations", "Organisation")
+    translation.activate("en")
```

Comment:
I wonder if we need to repeat this for all languages, I'm not very familiar with the `parler` package we use for the dynamic translations here

Reply:
I don't think we need to set the language here, it takes the default. the translated fields are saved in separate db tables I read in the docs. afaic we would not need to do it for all languages.

Reply:
If I remember correctly I added this line because it failed without it.

Reply:
I think we might have to talk to tj about what to do here, it's quite confusing. the ckeditor config allows iframe embeds, however the button isn't actually there to embed them, so in either case we probably don't need this migration because no one was able to add them. We just need to clarify if it should be possible in the new ckeditor

Reply:
@m4ra tj will join standup so we can talk about what to do here

Reply:
we can delete this migration